src/configuration/networks.js

```diff
@@ -21,7 +20,6 @@ export default new Map([
         ethereum: 'https://rinkeby.eth.aragon.network/',
       },
       orgLocation: 'agreement3.aragonid.eth',
-      disputableVotingApp: '0x26e14ed789b51b5b226d69a5d40f72dc2d0180fe',
```

Comment:
I removed this for now as we are waiting for the apps list before rendering the app and can get it dynamically.